langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -114,6 +159,13 @@ public isolated function indexOf(string str, string substr, int startIndex = 0)
 
 # Finds the last occurrence of one string in another string.
 #
+# ```ballerina
+# "Ballerinalang is a unique programming language".lastIndexOf("lang") ⇒ 38
+#
+# // Search backwards for the last occurrence of a string from a specific index.
```

Comment:
IMO, it would be better if we can have this comment inside this lang-lib function description.  WDYT?

Reply:
I don't want to change the description because it is coming from spec. But yes, I could do this when sending the PR to spec and get those changes back here.